test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/TestLoggerManagerTests.cs

```diff
@@ -1630,6 +1630,7 @@ private class InvalidLogger
     {
         public static int Counter = 0;
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "REVIEW ME: Is it ok to make this static?")]
```

Comment:
REVIEW ME.

Reply:
It should be, functionality-wise. Why do you think it would make difference?

Reply:
Please make sure it doesn't break compatibility across, I am still reviewing this but in general even if something is internal, if it crosses AppDomain boundaries we need to tread carefully.

Reply:
@Haplois do you have a concrete example where this caused a problem? We can go off on just "don't change things", but then the current state will never change.